src/api/app/assets/stylesheets/webui2/collapse-text.scss

```diff
@@ -0,0 +1,32 @@
+/*
+Copyright (c) 2019 by Natalia (https://codepen.io/natonischuk/pen/QbGWBa)
+Fork of an original work by Natalia (https://codepen.io/natonischuk/pen/mJOdmJ)
+Permission is hereby granted, free of charge, to any person obtaining a copy of
+this software and associated documentation files (the "Software"), to deal in the
+Software without restriction, including without limitation the rights to use, copy,
+modify, merge, publish, distribute, sublicense, and/or sell copies of the Software,
+and to permit persons to whom the Software is furnished to do so, subject to the
+following conditions: The above copyright notice and this permission notice shall be
+included in all copies or substantial portions of the Software. THE SOFTWARE IS PROVIDED
```

Comment:
As maintainer of cavil: Please format this like a MIT license is formatted. Aka: include some whitespace:

https://spdx.org/licenses/MIT.html#licenseText

Reply:
done

Reply:
This was the license they provide. I think that @krauselukas just copied it over

Reply:
The license doesn't change by formatting it nice